src/main.js

```diff
@@ -1,5 +1,127 @@
-'use strict'
+import JSONStream from 'JSONStream'
+import _ from 'underscore'
+import csv from 'csv-parser'
+import { unflatten } from 'flat'
+import highland from 'highland'
+import { SphereClient } from 'sphere-node-sdk'
 
-export default function addTwoNumbers (a, b) {
-  return a + b
+import CONSTANTS from './constants'
+import MapCustomFields from './map-custom-fields'
+
+export default class CsvParserPrice {
+  constructor (logger, apiClientConfig, config = {}) {
+    this.client = new SphereClient(apiClientConfig)
+    this.encoding = 'utf-8'
+    this.error = []
+    this.logger = logger
+    this.mapCustomFields = MapCustomFields()
+
+    this.config = config
+    this.batchProcessing =
```

Comment:
@Siilwyn Can we set all defaults in one object and use` _.defaults` to merge the two objects...